client/src/boot/BootRoutes.js

```diff
@@ -35,8 +36,7 @@ class BootRoutes {
     const base = Config.get('absoluteBaseUrl');
     pageRouter.setAbsoluteBase(base);
 
-    this.handleBeforeRoute = this.handleBeforeRoute.bind(this);
-    this.handleBeforeUnload = this.handleBeforeUnload.bind(this);
```

Comment:
Neither of these functions is passed into downstream components anymore, so we don't need explicit binds.